google-cloud-util/google-cloud-compat-checker/README.md

```diff
@@ -1,41 +0,0 @@
-Google Cloud Java Compatibility Checker
```

Comment:
Actually I don't want to ditch the google-cloud-compat-checker, people actually use it for debugging.